### ISSUE TITLE
package name differs from packagist name in composer.json

### ISSUE THREAD

**Issue by raffaelj** [agentejo/cockpit#1321]
In the [composer.json](https://github.com/agentejo/cockpit/blob/next/composer.json#L3), the name is `agentejo/cockpit`, but on packagist, the name is `aheinze/cockpit`.

So this one works and `lib/vendor/aheinze/cockpit` was created after running:
`composer install --no-dev --ignore-platform-reqs`

```json
{
    "name": "raffaelj/my-project",
    "require": {
        "aheinze/cockpit": "^0.11.0"
    }
}
```

But if I try to require a modified fork, the vendor name is wrong.

```json
{
    "name": "raffaelj/my-project",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/raffaelj/cockpit",
            "no-api": true
        }
    ],
    "require": {
        "aheinze/cockpit": "dev-wpmp-cockpit-lib"
    }
}
```

error: `Your requirements could not be resolved to an installable set of packages.`

If I change the vendor, to "agentejo", it works and the folder `lib/vender/agentejo/cockpit` is available with the content of my fork.

```json
{
    "name": "raffaelj/my-project",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/raffaelj/cockpit",
            "no-api": true
        }
    ],
    "require": {
        "agentejo/cockpit": "dev-wpmp-cockpit-lib"
    }
}
```

But now the folder name is different and the dependency has a wrong vendor name.

**Comment by aheinze** [comment]
Good point. but why would you install cockpit in a vendor folder? that's why I didn't updated the package. It was strange for me to install cockpit in a vendor folder

**Comment by raffaelj** [comment]
> but why would you install cockpit in a vendor folder?

I really like the concept of my [cockpit-lib-skeleton][1] with modified paths, a clean docs root and my own favicon.ico etc. So I can leave cockpit in a lib folder, I don't have to care about new files in the root, that aren't in the `.gitignore` file in a dev environment and I could update it with composer in a production environment.

In combination with [oomphinc/composer-installers-extender][2], I can install composer packages of type "project" in custom folders, e. g. `lib/cockpit` or `cockpit`, which makes composer a useful tool to build a project, that depends on cockpit, but has it's own base.

I also combined the concepts above in my latest work: [WPMultiplane][3] - a fully functional Cockpit CMS with CpMultiplane and a few cockpit addons packaged as an experimental WordPress plugin. I'm still not sure, if I created something beautiful or if it was a very, very bad idea - but it works.

I needed a modified fork of cockpit and now "agentejo/cockpit" is listed in the [dependency graph][4].

[1]: https://github.com/raffaelj/cockpit-lib-skeleton
[2]: https://github.com/oomphinc/composer-installers-extender
[3]: https://github.com/raffaelj/wordpress_wpmultiplane
[4]: https://github.com/raffaelj/wordpress_wpmultiplane/network/dependencies
